yelp.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 1.9.2'
+  spec.required_ruby_version = '>= 2.0.0'
```

Comment:
I'm fine with this, usage of < 2.0.0 is pretty small at this point in time. This was more of a problem when the gem was first made when 1.9.x was still popular. Let's make sure we bump the gem version appropriately when we release though